.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,153 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+  schedule:
+    - cron: "0 17 * * *"
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+
+    runs-on: "ubuntu-18.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2.0.0"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@1.8.2"
```

Comment:
i've said this before but i'm very much against this being used when docker containers are available.

Reply:
That's true, let's which direction we are taking here.

On the other hand, with Docker images we have the following issues:

- Docker images need to be mantained
- Docker images need to be built (either inline when running the job, which adds significant overhead, or built and tagged elsewhere, then downloaded)

I'm sure we'll be able to tweak things here and there!

Reply:
Hey!

Thanks for the reply.

I think I could say exactly the same about `shivammathur/setup-php` :)

- It is highly complex and maintenance is outside of our control here. We can either control Dockerfiles or use official php ones in most cases. I would trust official php docker images over `shivammathur/setup-php`.
- `shivammathur/setup-php` has significant overhead. It can take 25 seconds to install php, while pulling `php:7.4-cli-alpine3.11` takes only 6 seconds for example, in a quick test.
- As mentioned, i think the biggest thing is local reproducibility of build results.

Cheers!

Reply:
I am not sure one needs Docker to run this locally or? Its just php vendor/bin/phpcs, it doesn't have any dependencies or complexities to run.